src/Squirrel/ReleasePackage.cs

```diff
@@ -138,7 +138,7 @@ public string CreateReleasePackage(string outputFile, string packagesRootDir = n
             using (Utility.WithTempDirectory(out tempPath, null)) {
                 var tempDir = new DirectoryInfo(tempPath);
 
-                extractZipWithEscaping(InputPackageFile, tempPath).Wait();
+                extractZipWithEscaping(InputPackageFile, tempPath, x => { }).Wait();
```

Comment:
Can you please elaborate on this change?

Reply:
Yes, I've added comments.